### PR TITLE
create user modal and side nav

### PR DESCRIPTION
**Pull request by stephenv13**
- no actual user encryption implemented yet

### CODE REVIEW

File: components/admin/CreateUserModal.js [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+		// 	headers: new Headers({ "Content-Type": "application/json" }),
+		// 	body: JSON.stringify(data),
+		// });
+
+		resetValues();
+	};
+
+	return (
+		<>
+			<ListItemButton onClick={handleClickOpen}>
```
Comment (firehoops) [inline components/admin/CreateUserModal.js:96]: onClick={() => setOpen(!open)}

File: components/admin/CreateUserModal.js [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+									/>
+								</Grid>
+
+								{/* PASSWORD */}
+								<Grid item xs={12} sm={6}>
+									<TextField
+										required
+										autoFocus
+										margin="dense"
+										id="password"
```
Comment (firehoops) [inline components/admin/CreateUserModal.js:162]: id is the same here change to confirmPassword

File: components/admin/SideDrawer.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+import ListItemIcon from "@mui/material/ListItemIcon";
+import ListItemText from "@mui/material/ListItemText";
+import MenuIcon from "@mui/icons-material/Menu";
+import IconButton from "@mui/material/IconButton";
+import LogoutIcon from "@mui/icons-material/Logout";
+import { signOut } from "next-auth/react";
+import CreateUserModal from "./CreateUserModal";
+
+export default function SideDrawer() {
+	const [state, setState] = useState();
```
Comment (firehoops) [inline components/admin/SideDrawer.js:17]: openDrawer, also give a default value(false)

File: components/admin/CreateUserModal.js [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+import DialogActions from "@mui/material/DialogActions";
+import DialogContent from "@mui/material/DialogContent";
+import DialogContentText from "@mui/material/DialogContentText";
+import DialogTitle from "@mui/material/DialogTitle";
+import { Grid, Alert, AlertTitle, Snackbar, IconButton } from "@mui/material";
+import ListItemButton from "@mui/material/ListItemButton";
+import ListItemIcon from "@mui/material/ListItemIcon";
+import ListItemText from "@mui/material/ListItemText";
+import CloseIcon from "@mui/icons-material/Close";
+import PersonAddIcon from "@mui/icons-material/PersonAdd";
```
Comment (stephenv13) [inline components/admin/CreateUserModal.js:14]: move all imports to single line

**Review by firehoops — APPROVED** [review]

**Pull request merged**